exercise1.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+def generate_greeting(name):
```

Comment:
Perfect...this is doing _one_ thing: returning a string and _not_ dealing with output.